registers/dump/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/benlaurie/gds-registers/register"
+)
+
+var (
+	regname = flag.String("register", "register", "name of register (e.g. 'country')")
+)
+
+type dumper struct {
+}
+
+func (*dumper) Process(e map[string]interface{}, h string, i map[string]interface{}) error {
```

Comment:
Need a comment on the exported method (should this be exported? I guess not since it's in the `main` package?)

Reply:
Its part of the interface, so it needs to match, no?

Reply:
Ah, I didn't know it implemented an interface... maybe add a comment on the struct to that effect?